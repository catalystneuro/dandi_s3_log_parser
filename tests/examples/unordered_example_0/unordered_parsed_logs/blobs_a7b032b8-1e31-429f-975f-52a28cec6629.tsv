	timestamp	bytes_sent	region
0	2022-08-05 05:06:35	2141	CH/BeiJing
1	2022-03-21 05:06:35	24323	MX/Zacatecas
2	2019-11-13 05:06:35	4332423 US/Colorado
3	2020-04-16 05:06:35	12313153    unknown
