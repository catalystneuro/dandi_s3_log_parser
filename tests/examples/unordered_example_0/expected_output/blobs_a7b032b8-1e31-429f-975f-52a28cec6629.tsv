	timestamp	bytes_sent	region
0	2019-11-13 05:06:35	4332423 US/Colorado
1	2020-04-16 05:06:35	12313153    unknown
2	2022-03-21 05:06:35	24323	MX/Zacatecas
3	2022-08-05 05:06:35	2141	CH/BeiJing
